src/plugins/ui_feature/autofill.ts

```diff
@@ -222,13 +216,28 @@ export class AutofillPlugin extends UIPlugin {
           }
           const generator = this.createGenerator(xcs);
           for (let col = target.left; col <= target.right; col++) {
-            this.computeNewCell(generator, col, row, apply);
+            autofillCellsData.push(this.computeNewCell(generator, col, row));
           }
         }
         break;
     }
 
     if (apply) {
+      const borders: Record<UID, Border> = {};
```

Comment:
```suggestion
      const borders: Record<number, Border> = {};
```
if i'm not mistaken

Reply:
but can't we have a single `Record<StringifiedBorder, Zone[]>` instead of two different mappings for borders ? Or even just an array of `{zones: Zone[], border: Border}` and use this for the deepEquals ? I feel like it would be simpler than assigning a number ID to every different border